### ISSUE TITLE
log-D202312005

### ISSUE THREAD

**Issue by Linerre** [datenlord/training#17]
https://github.com/Linerre/mit

**Comment by Linerre** [comment]
Spring 2019  Version:
|Course|Date|What I did|Time spent|
|---|---|---|---|
|6.004|2023-12-22|Watched [Lec1 vid](https://youtu.be/n-YWa8hTdH8?si=W2AGL2xIbOKBLngW) & took notes|1h48m|
|6.004|2023-12-27|Watched [Lec2 vid](https://youtu.be/41RyDXIoq2w?si=QqNaC94f0XlZv8tc) & notes |1h35m|
|6.004|2024-01-11|Watched [Lec3 vid](https://youtu.be/GWSrieOIXls?si=CXDhrq2ox_6nQl7b) & notes |1h53m|

I'll record my lab progress in another comment of this thread later once I start working on labs of 6.175.

**Comment by myrfy001** [comment]
Good work!
 And which video lecture did you watch? As you said, there are a lot of versions and we recommend to watch 2019 Spring version.

By the way, since 6.004 doesn't have labs, so you can start 6.175's lab as you watch 6.004's video.

**Comment by Linerre** [comment]
> Good work! And which video lecture did you watch? As you said, there are a lot of versions and we recommend to watch 2019 Spring version.

Yes, I'm following the [Spring 2019 version](https://youtube.com/playlist?list=PLmQZXrtupiSYcziH3iZZoGTng1D0UnsFJ&si=zZUB_FXzXOCEF7ts) on Youtube.  I should make this clear in my notes and comments.

> By the way, since 6.004 doesn't have labs, so you can start 6.175's lab as you watch 6.004's video.

Ah I didn't even notice this.  The [6.191 schedule page](https://6191.mit.edu/fall23/schedule) lists lectures and labs and made me believe it has labs.  By "no labs", do you mean that the 6.004 lab content is _not_ available to non-mit students?  (It requires to log in) 

I just checked out the 6.175 labs and they are accessible easily, so I guess you suggest we should follow those open access labs.

PS: I just read through 6.175 course site and noticed it has the same instructor and textbook as 6.004 (6.191).  I'll start doing 6.175 labs as soon as possible.

**Comment by myrfy001** [comment]
> By "no labs", do you mean that the 6.004 lab content is _not_ available to non-mit students? (It requires to log in)

Yes, you're right.

**Comment by Linerre** [comment]
> I'll record my lab progress in another comment of this thread later once I start working on labs of 6.175.

| Course	| Date	| What I did	| Time spent|
|---|---|---|---|
| 6.175 |2024-01-11|Installed Bluespec|10min|
